src/Assertions.jl

```diff
@@ -125,11 +235,61 @@ macro v_do(s, l::Int, action)
   end
 end
 
+@doc Markdown.doc"""
+    set_verbose_level(s::Symbol, l::Int) -> Int
```

Comment:
Also rename this and other such functions? I.e.:
```suggestion
    set_verbosity_level(s::Symbol, l::Int) -> Int
```

Reply:
Well of course, I can also maybe change the `assert` to `assertion` in the similar way, if I transform the `verbose` into `verbosity` ? I will add some aliases for backwards compatibility (if used on Oscar)